CONTRIBUTING.md

```diff
@@ -35,152 +35,137 @@ The instance of helpbuttons runnning in https://dev.helpbuttons.org is up to dat
-
-copy the sample env file, and edit accordingly.
-
-`$ cp env.sample .env`
-
-> Note:
->
-> set `hostName` to `localhost`
->
-> set `VERSION` to `dev`
->
-> set `API_URL` to `http://localhost:3001/`
-
-generate jwt token:
-
-`$ docker-compose run api yarn config:genjwt`
-
-add the generated string as a `jwtSecret` to the `.env`
-
-Edit the `docker-compose.yml`, so that the frontend has access to the api, by binding the port 3001
-
-```
- ports: 
-  - 3001:3001
-```
-
-and make sure api is in the external_network network by uncomenting the line on the networks of the api
-
-` - external_network`
-
-You can run the api & database in docker you can do:
-
-`$ docker-compose up api`
-
-run all the migrations / setup the database schema:
-
-`$ docker-compose exec api yarn migration:run`
-
-to run the ui in development mode, enter the frontend source folder
-
-`$ cd web`
-
-create a symlink or copy the .env file to the api directory
-
-`$ ln -s ../.env .` or `$ cp ../.env .`
-
-install all node_modules packages
-
-`$ yarn`
-
-run the app in watch mode
-
-`$ yarn dev`
-
-You can now browse to `http://localhost:3000` to configure helpbuttons!
+1. Change to the dev branch
 
+`$ git checkout dev`
 
-## Setup frontend & api/backend for development
-
-We recommend that you use our pre-build docker image for postgres, because we added goodies like the [h3](https://github.com/uber/h3) library and gis.
-
-#### create .env file with your settings
-copy the sample env file, and edit accordingly.
+2. copy the sample env file
 
 `$ cp env.sample .env`
-> Note:
->
-> set `POSTGRES_HOSTNAME` to `localhost`.
->
-> set `hostName` to `localhost`
->
-> set `VERSION` to `dev`
->
-> set `API_URL` to `http://localhost:3001/`
-
-#### Setup database using docker
-You want to be able to access 5432 port on your localhost machine so you have to expose it. by editing the `docker-compose.yml` file.
-
-```
- ports: 
-  - 5432:5432
-```
-
-and make sure api is in the external_network network by uncomenting the line on the networks of the db
-
-` - external_network`
-
-run the database
-
-`$ docker-compose up db`
-
-#### Run the backend in development mode
-
-to run the api in watch mode you need firstly to go into the api directory
-
-`$ cd api`
-
-create the uploads directory and give the correct permissions
-
-`$ mkdir uploads`
-
-`$ chmod o+w uploads`
-
-install all node_modules packages
-
-`$ yarn`
-
-generate jwt token and add it to the .env file 
-
-`$ yarn config:genjwt`
-
-> add the generated string as a `jwtSecret` to the `.env` file
-
-create a symlink or copy the .env file to the api directory
-
-`$ ln -s ../.env .` or `$ cp ../.env .`
-
-run in watch/development mode
-
-`$ yarn dev`
-
-run all the migrations / setup the database schema:
-
-`$ yarn migration:run`
 
-to run the ui in development mode, enter the frontend source folder
+3. Choose one of the next two options:
+
++ <details>
+    <summary>API with Docker (to develope just in frontend)</summary>
```

Comment:
typo.. is not develope is develop :)

Reply:
also is not so clear.. maybe could be: 
I want to develop frontend code (api + db will run in docker)